.travis.yml

```diff
@@ -6,6 +6,10 @@ dist: xenial
 language: python
 cache: pip
 
+branches:
+  only:
+    - master
```

Comment:
Quick question @replaceafill but doesn't this then mean the dev branch won't be built when a PR is submitted?(Just raising this as the build seems stalled currently)

Reply:
@ross-spencer That's strange. It's the same change I made in the acceptance test repo: https://github.com/artefactual-labs/archivematica-acceptance-tests/pull/136/files

I'll restart the build and see if that fixes it.

Reply:
Yep. That did it.